### DISCUSSION TITLE
Different highlighting of context in backlinks

### DISCUSSION THREAD

**Discussion by Millesimus** [srid/neuron#666] [Ideas]
Hi! I really like the changes in the latest release, it also worked quite smoothly with my "productive" environment.

One thing I like in particular is the additional context given for backlinks. I would just ask a little help for my eyes to intuitively understand what is part of the context / quote from another zettel. I could imagine some typical quote highlighting (grey background, indentation, a vertical line to the left) instead of a bullet point would do the trick.

**Comment by srid** [comment]
Hey @Millesimus - I did something like that for Emanote, neuron's successor. See https://note.ema.srid.ca/resources/vscode for example.

Since neuron is in [bug-fix mode](https://neuron.zettel.page/next), you should look into Emanote.

**Reply by Millesimus** [reply]
Looks great, thanks!
